plugin/public/block.php

```diff
@@ -313,11 +334,11 @@ function build_result( $content, $type, $arg ) {
 	if ( 'list' === $type ) {
 		//build list table
 		if ( ! empty( $arg ) ) {
-			$result = '<table style="width:' . $arg['width'] . 'px"><tbody>';
-		} else {
-			$result = '<table><tbody>';
+			$style = ' style="width:' . parse_dimension( $arg['width'] ) . ';"';
 		}
 
+		$result = '<table class="sgdd-embedded-folder-list"' . $style . '><tbody>';
```

Comment:
Hi, I'm getting

```php
Notice: Undefined variable: style in /home/user/www/wp-content/plugins/skaut-google-drive-documents/public/block.php on line 340
```

Could you please look at it?

Reply:
What tool displays such message?

Reply:
Just the browser/server with `display_errors` turned on...

I think the error is on lines 328 and 336 which together may result in `$style` never being initialized when `$arg` is empty...

Reply:
Strange - I have `error_reporting=E_ALL` & `display_errors=On` and I don't see such notice.

Reply:
Is your `$arg` empty?

Reply:
Yes - I tested case with empty `$arg`. Anyway `$style = ''` should resolve this problem.